week-1/Homework/mandatory/3-function-output.js

```diff
@@ -1,14 +1,20 @@
 // Add comments to explain what this function does. You're meant to use Google!
+//This function returns a floating-point, pseudo-random number in the range 0 to less than 1 (inclusive of 0, but not 1) with approximately uniform distribution over that range — which you can then scale to your desired range.
 function getNumber() {
     return Math.random() * 10;
 }
 
 // Add comments to explain what this function does. You're meant to use Google!
+// This function concatenates the second string after the first string as seen below.
 function s(w1, w2) {
     return w1.concat(w2);
 }
 
 function concatenate(firstWord, secondWord, thirdWord) {
+
+    return firstWord.concat(` ${secondWord} ${thirdWord}`);
```

Comment:
This works but there is a better way to do this.

Reply:
Hi Abdo, here is a different way of scripting this part.